### DISCUSSION TITLE
scooty case, Machine learning. Error

### DISCUSSION THREAD

**Discussion by tatapinandhika** [teamalgoritma/community#151] [Q&A] [answered]
<img width="1440" alt="Screen Shot 2022-06-24 at 09 23 24" src="https://user-images.githubusercontent.com/101564223/175441510-28c536fe-a2f2-472e-9095-e2e6ab562592.png">
 mohon bantuan nya kakak tutor, saya error di bagian ini. 
Ijin Tag @fafilia

**Comment by tatapinandhika** [comment]
udah saya install library (purrr), tapi tetap error.

<img width="1440" alt="Screen Shot 2022-06-24 at 09 30 05" src="https://user-images.githubusercontent.com/101564223/175441767-0610d222-6908-44b1-894e-49fb0d4b9d2e.png">

**Comment by kevbow** [comment, marked as answer]
Halo @tatapinandhika,

Seperti error yang tertulis, parameter pertama pada fungsi map2 yang diminta harus berbentuk vector, sementara yang Bapak masukkan adalah function. Silakan baca kembali dokumentasi dari penggunaan fungsi tersebut pada link berikut https://www.rdocumentation.org/packages/purrr/versions/0.2.5/topics/map2.
